### PR TITLE
[[64-bit Mac]] Update the standalone builder and engine for slicing 32-/64-bit Universal binaries

### PR DESCRIPTION
**Pull request by livecodefraser**
There is a (soon-to-be submitted) PR to update the standalone settings stack in the IDE, too.

### CODE REVIEW

**Comment by livecodefraser** [comment]
Standalone settings UI change that makes these changes usable: https://github.com/livecode/livecode-ide/pull/810

File: ide-support/revsaveasstandalone.livecodescript [hunk 1]
```diff
@@ -616,13 +617,13 @@ private command revCreateStandalone pStack, pFolder
...
-         else
-            put revEnvironmentBinariesPath() into tBinariesPath
-         end if
+         --if tPlatform contains "MacOSX" and the platform is not "macos" or \
+               --tPlatform contains "Linux" and the platform is not "linux" or \
+               --tPlatform is "Windows" and the platform is not "win32" then
+         --put revEnvironmentNonNativeBinariesPath(tPlatform) into tBinariesPath
+         --else
+         put revEnvironmentBinariesPath() into tBinariesPath
+         --end if
```
Comment (peter-b) [inline ide-support/revsaveasstandalone.livecodescript]: Why comment this out rather than remove it?

Comment (livecodefraser) [inline ide-support/revsaveasstandalone.livecodescript]: Because it was committed accidentally :stuck_out_tongue:

**Comment by runrevmark** [comment]
@livecode-vulcan review ok d6880c5

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok d6880c540b1f6a6b90ce2c3e75db9ca5ad7d4aca

**Comment by livecode-vulcan** [comment]
😎 test success d6880c540b1f6a6b90ce2c3e75db9ca5ad7d4aca
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1681)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1697)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1676)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1687)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1698)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1700)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1705)

**Pull request merged**